bitnami/harbor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 3.0.7
+version: 3.0.8
```

Comment:
should we bump the minor instead?

Reply:
I usually follow this "approach":
> - MINOR version when you add functionality in a backwards-compatible manner
> - PATCH version when you make backwards-compatible bug fixes.

Why do you think it should be a minor?

Reply:
oh sorry. I felt like the values.yaml was moved around. That is not the case. You can go ahead with the merge